scripts/min_compatible_versions.py

```diff
@@ -0,0 +1,59 @@
+import csv
+import pathlib
+import sys
+from typing import Dict
+from typing import Set
+
+from packaging.version import parse as parse_version
+
+
+sys.path.append(str(pathlib.Path(__file__).parent.parent.resolve()))
+import riotfile  # noqa:E402
+
+
+OUT_FILENAME = "min_compatible_versions.csv"
+
+
+def _format_version_specifiers(spec: Set[str]) -> Set[str]:
+    return set([part.strip("~<>==") for v in [v.split(",") for v in spec if v] for part in v if "!=" not in part])
```

Comment:
how do we differentiate between `>3.0.0` and `>=3.0.0` ?

Reply:
we have some usage which is just `<2.0`, so that gives us an upper bound, not a lower bound.